client/a8c-for-agencies/components/slider/index.tsx

```diff
@@ -4,7 +4,7 @@ import './style.scss';
 
 export type Option = {
 	label: string;
-	value: number;
+	value: number | string | null;
```

Comment:
I missed considering this use case, but we must allow the value to be string (product slug) or null (more options).